addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -129,7 +130,11 @@ public MaxCubeBridgeHandler(Bridge br) {
     private boolean propertiesSet = false;
```

Comment:
Personally, I'd prefer to have each variable explicitly initialised, either in the constructor or as an argument.